maven-release.sh

```diff
@@ -65,6 +65,13 @@ sed_i "s/<artifactId>neo4j-connector-apache-spark_parent<\/artifactId>/<artifact
 sed_i "s/<artifactId>neo4j-connector-apache-spark_common<\/artifactId>/<artifactId>neo4j-connector-apache-spark_${SCALA_VERSION}_common<\/artifactId>/" "spark-3/pom.xml"
 sed_i "s/<artifactId>neo4j-connector-apache-spark_test-support<\/artifactId>/<artifactId>neo4j-connector-apache-spark_${SCALA_VERSION}_test-support<\/artifactId>/" "spark-3/pom.xml"
 
+LATEST_SCALA_VERSION="2.13"
```

Comment:
so Spark packages assume the latest-at-the-deploy-time Scala version?

Let's take an example:

- At t1, Scala latest is Scala 2.13 and Spark Connector `5.4.3` is released (without suffix)
- At t2>t1, Scala latest is now Scala 3. The `5.4.3` release is still for Scala 2.13, hopefully, right? If I release 5.4.4 now, it would be assumed that the release targets Scala 3?

Reply:
You are right. If we support multiple scala versions, we need to prefix all version. Will change it.